docs/guide/advanced/sfc.md

```diff
@@ -311,6 +401,99 @@ You can use define locale messages for global scope with `global` attribute:
 
 In the above example, since the `global` attribute is set, the locale messages defined in `i18n` custom blocks can be merged as a resource for locale messages of global scope.
 
+### Define global scope
+
+If you are using one of `@intlify/vite-plugin-vue-i18n`, `@intlify/vue-i18n-loader`  or `@intlify/rollup-plugin-vue-i18n`
```

Comment:
Same as the above my comment, we can use globalSFCScope and defaultSFCLocale options, so @intlify/rollup-plugin-vue-i18n only.
We need to mention at docs with :::warning:::